cha/implementaion/playback/meta.tex

```diff
@@ -0,0 +1,28 @@
+In this chapter we explore some key elements in the development of our application related to the playback of audio data.
+The chapter includes our decisions, the reasoning behind them, some of the complications we encountered and how we dealt with them.
+Before we present the implementation and what that entails, we firstly present some audio data terminology, as well as the Android sound stack, as this domain knowledge is used in our implementation.
+
+\begin{description}
+    \item [Sample Rate]\cite{sample_rate_std} \hfill \\
+    Sample rate is the number of audio samples per channel per second.
+    The rate may vary from file to file with $44,100\ Hz$ being the de facto and most supported sampling rate.
+    \item [Bit Depth]\cite{bit_depth} \hfill \\
+    Bit depth is largely dependant on the data encoding used, and if relevant also described alongside sample rate, e.g.\ a data rate of 44.1 kHz/16 translates to a sample rate of 44,100 and a bit depth of 16.
+    Bit depth is the number of bits used per sample.
+    In our project we work with MP3 and PCM encoding, and as such we are only concerned with these two formats.
+    MP3 does not encode any bit depth in its data, as such the bit depth for the \ac{PCM} data depends on the decoder settings.
+    As for PCM, this does contain bit depth data, and once again this may differ for each file with 16 and 24 bit both being common with 16-bit being the standard for CD, both Blu-ray and DVD also supports 24-bit.
+    \item [Channels]\cite{channels} \hfill \\
+    Channels refer to the number of audio tracks a file contains, this can be divided into three classes: Monaural Sound (Mono), Stereophonic Sound (Stereo) and Surround/Multichannel Sound.
```

Comment:
I grouped it like this because Mono and Stereo are relevant for us, thus i grouped the remaining, splitting it into quad, 5.1, 6.1, 7.1 would just be a worthless expansion imo.